src/style/toggle.js

```diff
@@ -48,7 +49,10 @@ const toggleStyle = {
     cursor: 'pointer',
     border: '1px solid rgb(220, 220, 220)',
     boxShadow: '0 1px 0px 0px rgb(185, 185, 185)',
-    transition: 'left 0.1s ease-in-out',
+
+    /* animations */
+    transition: 'left 0.1s, background-color 0.1s, boxShadow 0.1s, height 0.1s',
+    transitionTimingFunction: 'ease-in-out',
 
```

Comment:
Transition on toggle at times is giving me feeling that grab is not smooth any more.